.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,46 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [master, v1.x]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master, v1.x]
+  schedule:
+    - cron: '0 6 * * 3'
```

Comment:
not important, just curious why wed at 6am was selected?

Reply:
No idea; this was just the default. 😄